setup.cfg

```diff
@@ -23,8 +23,12 @@ url = https://pyaro.readthedocs.io
 [options]
 python_version = >=3.10
 install_requires =
-    numpy >= 1.13
+    numpy >= 1.13, <2.0.0
```

Comment:
Same error as we have seen previously in Pyaerocom AFAICT: https://github.com/metno/pyaro/actions/runs/10722641666/job/29734034686

Reply:
Ok, cf-units doesn't have new wheels with numpy2 compatibility. And since it is a binary, it doesn't work. I hope they will release a new one soon, last release was April 2023.